### PR TITLE
Added fixes to the kni1 environment

### PR DESCRIPTION
**Pull request by e-minguez**
Some changes related to the KNI1 environment.

### CODE REVIEW

**Comment by e-minguez** [comment]
@SchSeba PTAL thanks.

File: feature-configs/integration-kni1/ptp/subscription.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subscription
+  namespace: openshift-ptp
+spec:
+  channel: "4.4"
+  name: ptp-operator
+  source: "opsrctest"
```
Comment (fedepaol) [inline feature-configs/integration-kni1/ptp/subscription.yaml]: is this subscription correct?

**Comment by fedepaol** [comment]
Looks good, I'd like @SchSeba to give this a look as well

File: feature-configs/integration-kni1/ptp/subscription.yaml (hunk 1 above)
Comment (e-minguez) [reply feature-configs/integration-kni1/ptp/subscription.yaml]: I'm using the same one the demo uses here https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/demo/ptp/subscription.yaml#L10

Comment (fedepaol) [reply feature-configs/integration-kni1/ptp/subscription.yaml]: ah, I did not notice it! Thanks for explaining

File: feature-configs/integration-kni1/dpdk/kustomization.yaml [hunk 2]
```diff
@@ -3,3 +3,6 @@ kind: Kustomization
 
 bases:
   - ../../demo/dpdk
+
+patchesStrategicMerge:
+  - sriov-networknodepolicy-dpdk.yaml
```
Comment (SchSeba) [inline feature-configs/integration-kni1/dpdk/kustomization.yaml]: nit: please add an empty line here

File: feature-configs/integration-kni1/sctp/kustomization.yaml [hunk 3]
```diff
@@ -3,3 +3,7 @@ kind: Kustomization
 
 bases:
   - ../../base/sctp
+
+resources:
+  - sctp-demo-ns.yaml
+  - sctp-demo-svc.yaml
```
Comment (SchSeba) [inline feature-configs/integration-kni1/sctp/kustomization.yaml]: nit: please add an empty line here

File: feature-configs/integration-kni1/sctp/sctp-demo-svc.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    app: sctp-demo
+spec:
+  selector:
+    app: sctp-demo
+  ports:
+    - name: sctp-demo
+      protocol: SCTP
+      port: 30100
+      targetPort: 30100
+  type: ClusterIP
```
Comment (SchSeba) [inline feature-configs/integration-kni1/sctp/sctp-demo-svc.yaml]: nit: please add an empty line here

File: feature-configs/integration-kni1/sriov/sriov-subscription.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sriov-network-operator-subscription
+  namespace: openshift-sriov-network-operator
+spec:
+  channel: "4.4"
+  name: sriov-network-operator
+  source: opsrctest
+  sourceNamespace: openshift-marketplace
```
Comment (SchSeba) [inline feature-configs/integration-kni1/sriov/sriov-subscription.yaml]: nit: please add an empty line here

**Comment by e-minguez** [comment]
> some nits and one question.
> 
> Q: Can it be better to use the demo as the base for the Kustomize?

I've modified all features ti use demo as base except sriov because I'm still not using IPv6 (will do in a different environment).

PTAL

File: feature-configs/integration-kni1/dpdk/kustomization.yaml (hunk 2 above)
Comment (e-minguez) [reply feature-configs/integration-kni1/dpdk/kustomization.yaml]: Done, thanks!

File: feature-configs/integration-kni1/sctp/kustomization.yaml (hunk 3 above)
Comment (e-minguez) [reply feature-configs/integration-kni1/sctp/kustomization.yaml]: Done, thanks!

File: feature-configs/integration-kni1/sriov/sriov-subscription.yaml (hunk 5 above)
Comment (e-minguez) [reply feature-configs/integration-kni1/sriov/sriov-subscription.yaml]: Done, thanks!

File: feature-configs/integration-kni1/sctp/sctp-demo-svc.yaml (hunk 4 above)
Comment (e-minguez) [reply feature-configs/integration-kni1/sctp/sctp-demo-svc.yaml]: Done, thanks!

File: feature-configs/integration-kni1/sriov/sriov-networknodepolicy.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+metadata:
+  name: sriov-network-node-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames:
+    - eno1
+    rootDevices:
```
Comment (SchSeba) [inline feature-configs/integration-kni1/sriov/sriov-networknodepolicy.yaml]: you can remove this part the PfName is good

Comment (e-minguez) [reply feature-configs/integration-kni1/sriov/sriov-networknodepolicy.yaml]: you mean removing the `rootDevices`?

Comment (SchSeba) [reply feature-configs/integration-kni1/sriov/sriov-networknodepolicy.yaml]: yes there is no need for it :)

Comment (e-minguez) [reply feature-configs/integration-kni1/sriov/sriov-networknodepolicy.yaml]: Done, thanks!

**Pull request merged**